pkg/cmd/release/download/download.go

```diff
@@ -47,12 +51,15 @@ func NewCmdDownload(f *cmdutil.Factory, runF func(*DownloadOptions) error) *cobr
 		Example: heredoc.Doc(`
 			# download all assets from a specific release
 			$ gh release download v1.2.3
-			
+
 			# download only Debian packages for the latest release
 			$ gh release download --pattern '*.deb'
-			
+
 			# specify multiple file patterns
 			$ gh release download -p '*.deb' -p '*.rpm'
+
+			# download default zip package from a specific release
```

Comment:
Instead of saying `default * package` in the command docs, let's be consistent with the web UI where these archives are called "source code". How would this sound:
```suggestion
			# download the archive of the source code for a release
```